packages/yourloops/test/integration/mock/patient.api.mock.ts

```diff
@@ -33,8 +33,16 @@ import {
   pendingPatientAsTeamMember,
   patient2AsTeamMember
 } from '../data/patient.api.data'
+import { mySecondTeamId } from './team.api.mock'
 
 export const mockPatientApiForPatients = () => {
+  jest.spyOn(PatientApi, 'getPatient').mockImplementation((userId: string) => {
+    const patientsToReturn = PATIENTS_BY_TEAMID[mySecondTeamId]
+    if (!patientsToReturn) {
+      console.warn('Your mocked getPatient return is undefined, make sure that this is a wanted behaviour.', userId)
+    }
+    return Promise.resolve(patientsToReturn)
+  })
   jest.spyOn(PatientApi, 'getPatients').mockResolvedValue([patient1AsTeamMember])
   jest.spyOn(PatientApi, 'updatePatientAlerts').mockResolvedValue(undefined)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏I am a bit confused by this. Shouldn't it look more like this? 
```suggestion
  jest.spyOn(PatientApi, 'getPatient').mockImplementation((userId: string) => {
    const patientToReturn = PATIENTS.find(patient => patient.userid === userId)
    if (!patientToReturn) {
      console.warn('Your mocked getPatient return is undefined, make sure that this is a wanted behaviour.', userId)
    }
    return Promise.resolve([patientToReturn])
  })
  jest.spyOn(PatientApi, 'updatePatientAlerts').mockResolvedValue(undefined)
```